wallet/src/main/java/com/android/identity_credential/wallet/ui/prompt/passphrase/PassphrasePrompt.kt

```diff
@@ -47,18 +102,71 @@ suspend fun showPassphrasePrompt(
  * purposes of running [PassphrasePrompt] synchronously to return the prompt's result as
  * a [Boolean] and not worry about callback hell.
  *
+ * @param title the top-most text of the Prompt.
+ * @param content contains more information about the Prompt.
  * @param constraints the constraints for the passphrase.
- * @param checkWeakPassphrase if true, checks and disallows for weak passphrase/PINs and also
- *                            shows a hint if this is the case.
- * @param onPassphraseEntered callback issued from this dialog fragment when the user enters a passphrase
+ * @param onDismissDialog the callback invoked with a Boolean indicating whether onPassphraseResult() has
+ *      already been called which means the dismissal of the dialog is from the completion of the
+ *      Prompt rather than be dismissed because the user tapped outside of this dialog fragment.
+ * @param onPassphraseResult callback issued from this dialog fragment when the user enters a passphrase
  * @extends [BottomSheetDialogFragment] that can create the Fragment's contents via Composition.
  */
 class PassphrasePrompt(
```

Comment:
I think this can be a private class, no?

Reply:
Whenever it's set to be a private class, it compiles but trying to show the prompt yields to

```
java.lang.IllegalStateException: Fragment ...wallet.ui.prompt.passphrase.PassphrasePrompt 
must be a public static class to be  properly recreated from instance state.
```

The same for ConsentPrompt 
`ConsentPrompt must be a public static class to be  properly recreated from instance state.`